language/language.go

```diff
@@ -53,14 +53,21 @@ type Language interface {
 // Languages holds a register of all languages.
 var Languages = map[string]Language{}
 
+// LanguagesByFileExtension holds the a language file extension and the corresponding language.
```

Comment:
docstring typo... would call it `LanguageByFileExtension holds the language for a default file extension`.